pkg/fnenv/native/builtin/foreach.go

```diff
@@ -0,0 +1,82 @@
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/fission/fission-workflows/pkg/types"
+	"github.com/fission/fission-workflows/pkg/types/typedvalues"
+)
+
+const (
+	Foreach            = "foreach"
+	ForeachInputHeader = "foreach"
+	ForeachInputDo     = "do"
+)
+
+// FunctionForeach is a control flow construct to execute a certain task for each element in the provided input.
+// The tasks are executed in parallel. Foreach does not gather or store the outputs of the tasks in any way.
+//
+// For example:
+// foo:
```

Comment:
What will `$.Tasks.foo.Output` be?

Reply:
Same as above

Reply:
It is not done yet, but the idea was for foreach that there is no output (see description above this example). I am open to suggestions though :)